opensearch_py_ml/ml_commons/model_connector.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+from opensearchpy import OpenSearch
+
+from opensearch_py_ml.ml_commons.ml_common_utils import ML_BASE_URI
+
+
+class Connector:
+    def __init__(self, os_client: OpenSearch):
+        self.client = os_client
+
+    def create_standalone_connector(self, payload: dict):
+        if not isinstance(payload, dict):
+            raise ValueError("payload needs to be a dictionary")
```

Comment:
Can we add test related to this?

Reply:
Tests are acthally 100%. For somereason codecov is showing the stale results

Reply:
I don't think this is tested. you are testing with providing a dict. But here you need to provide something other than dict, so that a error raises and you are asserting that error, right?

Reply:
They are. Its actually issue with the codecov. its showing older report's warnings here